api/backend.go

```diff
@@ -298,21 +311,6 @@ func (b *StatusBackend) registerHandlers() error {
 		return b.AccountManager().Accounts()
 	})
 
-	rpcClient.RegisterHandler(params.SendTransactionMethodName, func(ctx context.Context, rpcParams ...interface{}) (interface{}, error) {
-		txArgs, err := transactions.RPCCalltoSendTxArgs(rpcParams...)
-		if err != nil {
-			return nil, err
-		}
-
-		hash, err := b.SendTransaction(ctx, txArgs)
-		if err != nil {
-			return nil, err
-		}
-
-		return hash.Hex(), err
-	})
-
-	rpcClient.RegisterHandler(params.PersonalSignMethodName, b.personalAPI.Sign)
```

Comment:
Similarly here.

Reply:
I don't think I understood your last point about creating a handler that returns an error if `eth_sendTransaction` is called. If we're keeping these then they should perform the expected actions and if we want to return an error for unrecognized/unavailable methods then we should have a separate handler for that no?

If we want to have these methods available via HTTP while playing with statusd then we should have them call `StatusBackend.SendTransaction/SignMessage` from the handler using the RPCParams.

Reply:
The problem is that we expect a password when sending a transaction. Default `eth_sendTransaction` method does not require the password in the params. I am not sure what's gonna happen without a handler but we definitely need to secure that.

Yeah, another option is to add one more param which would be the password. However, that also does not seem secure as the password should not be sent in HTTP requests.

That's why, in my opinion, it's easier to just write at the moment:
```go
rpcClient.RegisterHandler(params. SendTransactionMethodName, UnsupportedMethodHandler)
```